src/main.ts

```diff
@@ -1,23 +1,31 @@
 import { processTestResults } from './results';
-import { processTestCoverage } from './coverage';
-import { getInputs, publishComment, setFailed, setSummary } from './utils';
-import { formatCoverageMarkdown, formatResultMarkdown } from './formatting/markdown';
-import { formatCoverageHtml, formatResultHtml, formatTitleHtml } from './formatting/html';
+import { setFailed, setSummary } from './utils';
+import { formatResultMarkdown } from './formatting/markdown';
+import { formatResultHtml, formatTitleHtml } from './formatting/html';
+import * as fs from 'fs';
 
 const run = async (): Promise<void> => {
   try {
-    const {
-      token,
-      title,
-      resultsPath,
-      coveragePath,
-      coverageType,
-      coverageThreshold,
-      postNewComment,
-      allowFailedTests,
-      onlyShowFailedTests,
-      showTestOutput
-    } = getInputs();
+    // const {
+    //   token,
+    //   title,
+    //   resultsPath,
+    //   coveragePath,
+    //   coverageType,
+    //   coverageThreshold,
+    //   postNewComment,
+    //   allowFailedTests,
+    //   onlyShowFailedTests,
+    //   showTestOutput
+    // } = getInputs();
```

Comment:
You forgot some test code here. Also please pull the latest main, regenerate package lock (`npm install`) and regenerate build files (`npm run build`)

Reply:
Sorry that I missed that, hopefully the last one